packages/react/src/components/dateInput/__snapshots__/DateInput.test.tsx.snap

```diff
@@ -2583,6 +2583,7 @@ exports[`<DateInput /> spec renders the component with default props 1`] = `
               class="hds-datepicker__bottom-buttons"
             >
               <button
+                aria-disabled="true"
```

Comment:
Are these changes wanted?

Reply:
I quickly checked the datepicker code and it seems that these button were disabled